collect_app/src/main/res/layout-v21/toolbar_action_bar_shadow.xml

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We need this file it's for api 21+. You just need to move it to the main layout package to replace the one for other APIs.